maps/packedstation/uplink_item.dm

```diff
@@ -0,0 +1,29 @@
+// Modified uplink items to discourage tator murderboning
```

Comment:
Welp my bad, that was inside of the ministation things, i forgot to comment it like the jobs.dm inside the folder